presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -62,22 +66,22 @@ public void runCommand()
                 // TODO: Add tarball installation
                 throw new UnsupportedOperationException("Tarball installation is not supported");
             default:
-                // TODO: Add to logger
                 throw new IllegalArgumentException(format("Unsupported package type %s", packageType));
         }
     }
 
     private static void installUsingRpm(String pathToRpm, boolean checkDependencies)
             throws PrestoManagerException
     {
-        String checkRpm = format("rpm -K --nosignature %s", pathToRpm);
-        if (executeCommand(checkRpm) != 0) {
-            throw new PrestoManagerException("Corrupted RPM");
+        int checkRpm = executeCommand(format("rpm -Kv --nosignature %s", pathToRpm));
+        if (checkRpm != 0) {
+            throw new PrestoManagerException("Corrupted RPM", checkRpm);
         }
         String nodeps = checkDependencies ? "" : "--nodeps";
-        String installRpm = format("sudo rpm -i %s %s", nodeps, pathToRpm);
-        if (executeCommand(installRpm) != 0) {
-            throw new PrestoManagerException("Failed to install presto");
+        int installRpm = executeCommand(format("sudo rpm -iv %s %s", nodeps, pathToRpm));
```

Comment:
`pathToRpm` may cause problems if it has an unusual name.

The most important reason to pass the arguments to a subprocess as separate strings is because of this sort of thing.

Imagine if the path provided was `--prefix=/sbin init.rpm`.